lib/solar_system.rb

```diff
@@ -0,0 +1,29 @@
+class SolarSystem
+  attr_reader :planets
+
+  def initialize (star_name)
+    @star_name = star_name
+    @planets = [] 
+  end
+
+  def add_planets(some_planet)
+    @planets << some_planet
+  end
+
+  def list_planets
+    sentence_array = []
+    opening_sentence = "Planets orbiting the #{@star_name}: "
+    sentence_array << opening_sentence
+    @planets.each_with_index do | planet, index|
+      listing_planets = "#{index + 1}: #{planet.name}"
+      sentence_array << listing_planets
+    end
+    return sentence_array
+  end
+
+  def find_planet_by_name (planet_name)
+    return planet_name
+  end
```

Comment:
This is not what we were looking for with this method. Our expectation was that you would search through the list of planets and find one that matches the parameter. Something like this:

```ruby
def find_planet_by_name(planet_name)
  @planets.each do |planet|
    return planet if planet.name == planet_name
  end
  # no matching planet found
  return nil
end
```

As-is, all this method does is return whatever the caller passed in, which is not very useful.